docs/mara_lampert/getting_started_with_mambaforge_and_python/readme.md

```diff
@@ -101,7 +101,7 @@ The reason for this is that incompatibilities between packages can occur. Robert
 You can create a new environment typing the following command into the Command Prompt: 
 
 ```json
-mamba create -n my_first_env devbio-napari python=3.9 -c conda-forge 
+mamba create --name devbio-napari-env python=3.9 devbio-napari pyqt -c conda-forge -c pytorch
```

Comment:
```suggestion
mamba create -n my_first_env devbio-napari python=3.9 pyqt -c conda-forge
```
I'm changing the line to minimal conflicts with other commands in the blog post. E.g. we should not change the name of the environment.